packages/styles/popover.css

```
@@ -0,0 +1,116 @@
+:root {
+  --popover-background-color: var(--gray-10);
+  --popover-border-color: var(--gray-90);
+  --popover-text-color: var(--gray-90);
+  --popover-box-shadow: rgba(0, 0, 0, 0.35) 0px 0px 10px 0px;
```

Very minor nit: 0 values can be unitless in css.

```suggestion
  --popover-box-shadow: rgba(0, 0, 0, 0.35) 0 0 10px 0;
```